### PR TITLE
Added revocation service, reworked documentation, updated dependencies

### PR DESCRIPTION
**Pull request by ghost**
After the IP approval of managed identity wallet, this is the pull request to update the Eclipse repository with the latest features from the Catena-X repository and prepare the developer move to the Eclipse repository.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [2.1.1] - 2022-10-07
+
+### Added
+- Wallet handling via CRUD
```
Comment (jpbu) [inline CHANGELOG.md:12]: Should mention/distinguish managed wallets vs. self-managed wallets (limited support for the latter)

File: Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM amazoncorretto:17-alpine
+FROM eclipse-temurin:19-jre-alpine
```
Comment (jpbu) [inline Dockerfile:1]: I second to switch to Eclipse Temurin, but I would stick to Java 17 (LTS) - did you test 19?

**Review by jpbu — COMMENTED** [review]
Looks good to me, two inline comments.

I think we need to be careful once this is merged from develop to main, since the main branch already seems to contain the renaming CatenaX to Eclipse.

**Comment by MajdT51** [comment]
Looks good to me.

File: Dockerfile (hunk 2 above)
Comment (ghost) [reply Dockerfile:1]: Yes we did: the test cases ran through, also we've deployed it on the dev environment and ran it successfully for a few weeks already

File: CHANGELOG.md (hunk 1 above)
Comment (ghost) [reply CHANGELOG.md:12]: Good point, thanks for pointing out, I will clarify that in the next version

**Pull request merged**